canmatrix/kcd.py

```diff
@@ -79,9 +77,7 @@ def createSignal(signal, nodeList, typeEnums):
             consumer.append(noderef)
         if consumer.__len__() > 0:
             sig.append(consumer)
-
-
-    value = etree.Element('Value')
+   value = etree.Element('Value')
```

Comment:
semms that indentation is wrong:
https://travis-ci.org/ebroecker/canmatrix/jobs/407752817#L480